src/main/java/com/vonage/quickstart/application/CreateApplication.java

```diff
@@ -32,7 +32,7 @@
 import static com.vonage.quickstart.Util.envVar;
 
 public class CreateApplication {
-    private static final String NAME = "Code Snippets V2 Application";
+    private static final String APPLIACATION_NAME = "Code Snippets V2 Application";
```

Comment:
Typo.

But shouldn't this not be hard-coded but instead be passed from the request?

Reply:
fixed